File: NetEase_Modular/common/src/main/java/com/netease/common/base/BaseActivity.java
package com.netease.common.base;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v7.app.AppCompatActivity;
import android.util.Log;

import com.netease.common.utils.Cons;

/**
 * Create by 陈扬齐
 * Create on 2019-07-15
 * description:
 */
public class BaseActivity extends AppCompatActivity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        Log.e(Cons.TAG, "common/BaseActivity");
    }
}

File: ModuleDemo/common/src/main/java/com/cyq/common/base/BaseActivity.kt
package com.cyq.common.base

import android.os.Bundle
import android.os.PersistableBundle
import androidx.appcompat.app.AppCompatActivity

/**
 *    @author : ChenYangQi
 *    date   : 2020/10/14 15:55
 *    desc   :
 */
open class BaseActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?, persistentState: PersistableBundle?) {
        super.onCreate(savedInstanceState, persistentState)

    }
}